pkg/storage/azure/azure.go

```diff
@@ -675,7 +678,7 @@ func (d *driver) assurePrivateAccount(cfg *Azure, infra *configv1.Infrastructure
 	}
 
 	if internalConfig.VNetName == "" {
-		tagKey := fmt.Sprintf("kubernetes.io_cluster.%s", infra.Status.InfrastructureName)
+		tagKey := fmt.Sprintf(vnetTagKeyTemplate, infra.Status.InfrastructureName)
```

Comment:
The advice from installer folks is to look for both tags. I'll update this code tomorrow.